### ISSUE TITLE
Administration landing page missing H1

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#2386]
There should be a visually hidden H1 on the administration  landing page:

![image.png](https://images.zenhubusercontent.com/620427f3846ebb620d78e582/bc0e5aae-48a0-4aa0-bff2-53b6cc4f4a07)

![image.png](https://images.zenhubusercontent.com/620427f3846ebb620d78e582/77305381-fe39-45d1-bb29-3ad0b9a1ebc1)
